### DISCUSSION TITLE
Export overlays with transparent image

### DISCUSSION THREAD

**Discussion by puntiz** [hm21/pro_image_editor#189] [Q&A] [answered]
Is it possible to export as an png image just the text / sticker / emoji overlays on a transparent canvas rather than the source image ? 

I understand that filters / crops cannot be exported and image size ratio needs to be maintained.

Thanks again for the amazing package

**Comment by hm21** [comment, marked as answer]
Hi,

Yes, it's possible to do that. You can follow this [example](https://github.com/hm21/pro_image_editor/blob/stable/example/lib/pages/movable_background_image.dart), which shows you how to do it. The part in the onAfterViewInit callback is not necessary if you don't want a background image, so you can remove it.
